src/views/FfaListedView.vue

```diff
@@ -242,10 +244,21 @@ export default class FfaListedView extends Vue {
       },
       label: this.details,
     })
-
     this.$root.$on(DrawerClosed, this.onDrawerClosed)
-    this.$root.$on(ChallengeResolved, this.postResolveChallenge)
-    this.$store.subscribe(this.vuexSubscriptions)
+    this.unsubscribe = this.$store.subscribe(this.vuexSubscriptions)
+  }
+
+  public async mounted(this: FfaListedView) {
+
+    if (this.$router.currentRoute.name === 'singleListed') {
+      this.$root.$emit(CloseDrawer)
+
+}
+    this.votingModule.reset()
```

Comment:
`this.votingModule.reset()` in `created` and `mounted`

Reply:
Do you mean should we remove one since one is redundant?